src/bots/coverbot/index.ts

```diff
@@ -135,10 +146,50 @@ export class Coverbot implements Bot {
         this.verifiedHoprNodes = new Map<string, HoprNode>()
         connected.forEach((n) => this.verifiedHoprNodes.set(n.id, n))
         log(`- loadData | Updated ${Array.from(this.verifiedHoprNodes.values()).length} verified nodes in memory`)
-        this.loadedDb = true
         return resolve()
       })
     })
+
+    // restore scores only if 'COVERBOT_RESTORE_SCORE_FROM' is provided
+    if (COVERBOT_RESTORE_SCORE_FROM) {
```

Comment:
And also if `score` is empty, otherwise we might overwrite a database that’s already being written on

Reply:
Ah, didn’t see the check you did below, sorry